File: app/src/main/java/nolambda/stiker/motionviews/widget/MotionViewCallback.java
package nolambda.stiker.motionviews.widget;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import nolambda.stiker.motionviews.widget.entity.MotionEntity;

public interface MotionViewCallback {
    void onEntitySelected(@Nullable MotionEntity entity);

    void onEntityDoubleTap(@NonNull MotionEntity entity);
}

File: app/src/main/java/nolambda/stiker/motionviews/widget/MotionViewCallback.kt
package nolambda.stiker.motionviews.widget

import nolambda.stiker.motionviews.widget.entity.MotionEntity

interface MotionViewCallback {
    fun onEntitySelected(entity: MotionEntity?)
    fun onEntityDoubleTap(entity: MotionEntity)
}
